### PR TITLE
Update k8s-kvm container image for WIP branches

### PR DESCRIPTION
**Pull request by tuommaki**
New image contains polling for `shutdown-deferrer` service: https://github.com/giantswarm/k8s-kvm/pull/23

Now `shutdown-deferrer` is also bound to port `60080` to avoid conflicts.

### CODE REVIEW

**Comment by tuommaki** [comment]
As noted with referenced `k8s-kvm` PR, this has been tested on `gorgoth` with current active, current WIP and upgrade from current active to WIP.

File: service/controller/v15/resource/deployment/master_deployment.go [hunk 1]
```diff
@@ -325,6 +325,11 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 								Name:            "shutdown-deferrer",
 								Image:           key.ShutdownDeferrerDocker,
 								ImagePullPolicy: apiv1.PullAlways,
+								Command: []string{
```
Comment (xh3b4sd) [inline service/controller/v15/resource/deployment/master_deployment.go]: Can't we use args or something? The entrypoint should already be the binary and daemon command.

**Review by xh3b4sd — APPROVED** [review]
When this works lets go. I was wondering if we should sort some health checks or something. What happens when the deferrer fails for some reason?

**Comment by tuommaki** [comment]
> I was wondering if we should sort some health checks or something.

Yes. I was planning to add this later, but this PR works well for that. I'll sort this out.

**Comment by tuommaki** [comment]
Successfully tested [latest version] on `gorgoth`.

**Pull request merged**